fullstack-projects/notes-app/devashmit-day19/backend/main.py

```
@@ -0,0 +1,47 @@
+    with database.get_conn() as c:return row(get404(nid,c))
+
+@app.post("/notes",response_model=NO,status_code=201)
+def create(b:NC):
+    with database.get_conn() as c:
+        cur=c.execute("INSERT INTO notes(title,content) VALUES(?,?)",(b.title,b.content or ""))
+        return row(c.execute("SELECT * FROM notes WHERE id=?",(cur.lastrowid,)).fetchone())
+
+@app.put("/notes/{nid}",response_model=NO)
+def update(nid:int,b:NU):
+    with database.get_conn() as c:
+        get404(nid,c);u=b.model_dump(exclude_unset=True)
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)